components/servo/webview.rs

```diff
@@ -310,68 +309,30 @@ impl WebView {
             ))
     }
 
-    pub fn notify_pointer_button_event(&self, event: MouseWindowEvent) {
-        self.inner()
-            .compositor
-            .borrow_mut()
-            .on_mouse_window_event_class(event);
-    }
-
-    pub fn notify_pointer_move_event(&self, event: DevicePoint) {
-        self.inner()
-            .compositor
-            .borrow_mut()
-            .on_mouse_window_move_event_class(event);
-    }
-
-    pub fn notify_touch_event(&self, event_type: TouchEventType, id: TouchId, point: DevicePoint) {
-        self.inner()
-            .compositor
-            .borrow_mut()
-            .on_touch_event(event_type, id, point);
-    }
-
-    pub fn notify_wheel_event(&self, delta: WheelDelta, point: DevicePoint) {
-        self.inner()
-            .compositor
-            .borrow_mut()
-            .on_wheel_event(delta, point);
-    }
-
```

Comment:
Now there is just `WebView::notify_input_event`. I haven't touched `notify_scroll_event` yet, as I suspect that this will turned into an API like `scroll()`.